src/game/camera.c

```diff
@@ -871,9 +894,30 @@ s32 update_8_directions_camera(struct Camera *c, Vec3f focus, Vec3f pos) {
     s16 pitch = look_down_slopes(camYaw);
     f32 posY;
     f32 focusY;
+#ifdef REONUCAM
+    f32 yOff;
+#else
     f32 yOff = 125.f;
+#endif
     f32 baseDist = 1000.f;
 
+#ifdef REONUCAM
+    if (gMarioState->action & ACT_FLAG_SWIMMING) {
+        yOff = -125.f;
+    } else {
+        yOff = 125.f;
+    }
```

Comment:
default to 125 like how it is, then override it if swimming

```suggestion
    f32 yOff = 125.f;
    f32 baseDist = 1000.f;

#ifdef REONUCAM
    if (gMarioState->action & ACT_FLAG_SWIMMING) {
        yOff = -125.f;
    }
```